draft-ietf-quic-transport.md

```diff
@@ -4747,9 +4747,10 @@ QUIC acknowledgements are irrevocable.  Once acknowledged, a packet remains
 acknowledged, even if it does not appear in a future ACK frame.  This is unlike
 TCP SACKs ({{?RFC2018}}).
 
-An ACK frame acknowledges packets in the same packet number space as the packet
-in which it is contained.  As such, the same packet number can be acknowledged
-in different packet number spaces.
+The same packet number might be used to identify packets from multiple packet
+number spaces. An acknowledgement for a packet therefore needs to indicate both
+its packet number and its packet number space.  An ACK frame acknowledges packet
+numbers in the same space as the packet in which the ACK frame is contained.
```

Comment:
These three sentences are kind of hard to connect.

Perhaps:
"An acknowledgment for a packet needs to indicate both its packet number and its packet number space, which is accomplished by having each ACK frame only acknowledge packet numbers..."